libs/common-services/src/test/java/uk/gov/di/ipv/core/library/service/CiMitUtilityServiceTest.java

```diff
@@ -286,8 +287,9 @@ void getMitigationJourneyResponseShouldReturnEmptyWhenMitigationDoesNotResolveBr
     }
 
     @Test
-    void getMitigationJourneyResponseShouldThrowWhenCiMitigationJourneyConfigNotFoundForDocType()
-            throws Exception {
+    void
+            getCiMitigationJourneyStepShouldReturnEmptyOptionalWhenCiMitigationJourneyConfigNotFoundForDocType()
```

Comment:
(If you want this on one line, I'd be happy with `...ShouldReturnEmptyWhen...`)

Reply:
Spotless still loves a newline with the change...